examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
I basically left the code in the `mpl.show(mpl.to_bokeh(...))` form to avoid deeper modifications at this point. Just to note, `mpl.show()` is `plotting.show()` thanks to Python's "magic" imports. I presume I should add explicit import of `show()` in those examples and replace `mpl.show` with `show`?

> Finally using mpl.show(mpl... is too much mpl-verbose

Perhaphs, but at least it tells the right story.